core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -858,6 +859,7 @@ public function loadDetails()
 		$this->groups = $this->protected ? StringUtil::deserialize($this->groups) : false;
 		$this->layout = $this->includeLayout ? $this->layout : false;
 		$this->cache = $this->includeCache ? $this->cache : false;
+		$this->forceCache = $this->forceCache ? $this->forceCache : false;
```

Comment:
`(bool) $this->forceCache`

Reply:
True fixed in 430dda656dcef832125c1ae6c0b7fa6a231e641d.